cpp/src/parquet/schema-internal.h

```diff
@@ -37,6 +39,57 @@ class SchemaElement;
 
 namespace schema {
 
+inline bool str_endswith_tuple(const std::string& str) {
```

Comment:
Only short-running performance-critical functions should be inlined. Is it necessary here?

Reply:
This is just moving code from parquet/util/schema-util.h